apiary.apib

```diff
@@ -690,6 +692,12 @@ If you receive a network timeout or other unexpected response (such as a 5xx) wh
 ## Funds [/funds]
 Real-time information on your account funding.
```

Comment:
Should this be a bit more rounded out to describe that it is the 'GET'?  Seems a bit to fragment-y with all the info below on funding